.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,38 @@
 
+---
+**PR checklist**
+
+The following checklist shall help the PR's author, the reviewers and maintainers to ensure the quality of this project.
+It bases on our contributors guidelines, especially the ["writing code" section](https://github.com/junit-pioneer/junit-pioneer/blob/master/CONTRIBUTING.md#writing-code).
+It shall help to check for completion of the listed points.
```

Comment:
Simplify: `It helps check for the completion of the listed points`

Reply:
See above: Shall used by purpose